cmake/finders/FindLibspeexdsp.cmake

```diff
@@ -124,14 +125,16 @@ if(Libspeexdsp_FOUND)
     libspeexdsp_set_soname()
     set_target_properties(
       SpeexDSP::Libspeexdsp
-      PROPERTIES INTERFACE_COMPILE_OPTIONS "${PC_Libspeexdsp_CFLAGS_OTHER}"
-                 INTERFACE_INCLUDE_DIRECTORIES "${Libspeexdsp_INCLUDE_DIR}"
-                 VERSION ${Libspeexdsp_VERSION})
+      PROPERTIES
+        INTERFACE_COMPILE_OPTIONS "${PC_Libspeexdsp_CFLAGS_OTHER}"
+        INTERFACE_INCLUDE_DIRECTORIES "${Libspeexdsp_INCLUDE_DIR}"
+        VERSION ${Libspeexdsp_VERSION}
+    )
   endif()
 endif()
 
 include(FeatureSummary)
 set_package_properties(
-  Libspeexdsp PROPERTIES
-  URL "https://gitlab.xiph.org/xiph/speexdsp"
-  DESCRIPTION "DSP library derived from speex.")
+  Libspeexdsp
+  PROPERTIES URL "https://gitlab.xiph.org/xiph/speexdsp" DESCRIPTION "DSP library derived from speex."
```

Comment:
This change is surprising, but I'm not going to block on it or ask you to suppress it.

Reply:
I think the rules are quite simple: It either fits onto one line, or - if it doesn't - it will be broken up and then function arguments will be put on separate lines.

Reply:
I arrived at that supposition after the fact, but wanted confirmation nonetheless.